source/client-side-encryption/client-side-encryption.rst

```diff
@@ -929,6 +929,7 @@ listIndexes            BYPASS
 renameCollection       BYPASS
 explain                AUTOENCRYPT
 ping                   BYPASS
+killAllSessions        BYPASS
```

Comment:
Do we also want to add the other sessions related commands: https://docs.mongodb.com/manual/reference/command/nav-sessions/
- killSessions
- killAllSessionsByPattern
- refreshSessions

Reply:
Ooh, I was unaware of those. Seems reasonable to bypass them as well. Updated.